TypeAdapters.kt

```diff
@@ -0,0 +1,78 @@
+package pm.gnosis.eip712.adapters.moshi
```

Comment:
So this file should probably be removed from this module then?

Reply:
Ouch. Yes.

Reply:
Fixed in a63f0af36cafc2fa3861eaa5d0e24dacb35ffc31